conf/exports-wco-mapping.json

```diff
@@ -0,0 +1,402 @@
+[
+  {
+    "cds" : "consignmentReferences.lrn",
+    "wco" : "42A.D026"
+  },
+  {
+    "cds" : "goodsItemQuantity",
+    "wco" :  "42A.228"
+  },
+  {
+    "cds" : "items.$1",
+    "wco" : "42A.67A.68A.$1"
+  },
+  {
+    "cds" : "items.$1.additionalDocuments.documents.$2",
+    "wco" : "42A.67A.68A.$1.02A.$2"
+  },
+  {
+    "cds" : "items.$1.additionalFiscalReferencesData.references.$2",
+    "wco" : "42A.67A.68A.$1.55B.$2"
+  },
+  {
+    "cds" : "items.$1.additionalInformation.items.$2",
+    "wco" : "42A.67A.68A.$1.03A.$2"
+  },
+  {
+    "cds" : "items.$1.commodityDetails.combinedNomenclatureCode",
+    "wco" : "42A.67A.68A.$1.23A"
+  },
+  {
+    "cds" : "items.$1.commodityDetails.descriptionOfGoods",
+    "wco" : "42A.67A.68A.$1.23A.137"
+  },
+  {
+    "cds" : "items.$1.commodityMeasure.grossMass",
+    "wco" : "42A.67A.68A.$1.23A.65A.126"
+  },
+  {
+    "cds" : "items.$1.commodityMeasure.netMass",
+    "wco" : "42A.67A.68A.$1.23A.65A.128"
+  },
+  {
+    "cds" : "items.$1.commodityMeasure.supplementaryUnits",
+    "wco" : "42A.67A.68A.$1.23A.65A.130"
+  },
+  {
+    "cds" : "items.$1.nactCode.$2",
```

Comment:
Can we add a comment for this item and the next one (taric) to the effect of "CDS pointer refers to one in a sequence of codes, but WCO pointer points to the parent level element"